lib/stats/series.js

```diff
@@ -0,0 +1,195 @@
+  })
+});
+
+export const workerSpotTermination = new base.stats.Series({
+  name: 'worker_shutdown',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time node termination event emitted
+    value: base.stats.types.Number
+  })
+});
+
+export const volumeCacheHit = new base.stats.Series({
+  name: 'volume_cache_hit',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Cache name
+    value: base.stats.types.String
+  })
+});
+
+export const volumeCacheMiss = new base.stats.Series({
+  name: 'volume_cache_miss',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Cache name
+    value: base.stats.types.String
+  })
+});
+
+export const timeToFirstClaim = new base.stats.Series({
+  name: 'tasks_first_claim',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time in ms from when the task was created to when the first worker claimed
+    // it
+    value: base.stats.types.Number
+  })
+});
+
+export const claimTask = new base.stats.Series({
+  name: 'tasks_claim',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const abortTask = new base.stats.Series({
+  name: 'tasks_abort',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const reclaimTask = new base.stats.Series({
+  name: 'tasks_reclaim',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const cancelTask = new base.stats.Series({
+  name: 'tasks_cancel',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const taskError = new base.stats.Series({
+  name: 'tasks_error',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const runTimeExceeded = new base.stats.Series({
+  name: 'tasks_run_time_exceeded',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const taskRunTime = new base.stats.Series({
+  name: 'tasks_runtime',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const taskFeature = new base.stats.Series({
+  name: 'task_feature',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Feature name
+    value: base.stats.types.String
+  })
+});
+
+export const taskImage = new base.stats.Series({
+  name: 'task_image',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Image name
+    value: base.stats.types.String
+  })
+});
+
+export const devicePhone = new base.stats.Series({
+  name: 'task_device_phone',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    type: base.stats.types.String,
+    sims: base.stats.types.String
+  })
+});
+
+export const statesLink = new base.stats.Series({
+  name: 'states_linked',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
```

Comment:
Please, let's merge these into one series...

And lets call `value` for `duration`?

I see one series called `stateChanged` with columns `duration` and `state` as string (later tag in 0.9)..